install.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+install_path=${1:-./uwazi-production/}
+install_path=${install_path%/}
+
+echo "Building production"
+yarn production-build
+
+echo installing into $install_path
+mkdir -p $install_path
+mkdir -p $install_path/custom_uploads
+mkdir -p $install_path/log
+mkdir -p $install_path/temporal_files
+mkdir -p $install_path/uploaded_documents
```

Comment:
This is not respecting the env variable UPLOADS_FOLDER which is described as configurable on the custom configuration description.  https://github.com/huridocs/uwazi/wiki/Install-Uwazi-on-your-server

Reply:
I guess is the same for temporal_files and uploaded_documents. Why are you pointing out this one particularly? In the configuration description is using the env var UPLOADS_FOLDER, but this is used for uploaded_documents, which is in this script. Can you explain what is the problem?

Reply:
@RafaPolit awaiting your clarification to merge this.

Reply:
Yes, perhaps there is no problem here.  It will work as expected, yes.